src/templates/_macros/addon_tile.html

```diff
@@ -1,5 +1,24 @@
-{% macro addon_tile(addon) %}
-  <article class="addon">
-    <h1><a href="{{ url('addon', [addon.slug]) }}">{{ addon.name }}</a></h1>
-  </article>
+{% macro addon_tile(addon, is_detail=False) %}
+{% set tag = 'div' if is_detail else 'a' %}
+  <{{ tag }} class="mkt-tile"
```

Comment:
let's not use mkt-tile. this causes addon and app to share styles which will introduce regressions later